doc/api/http2.md

```diff
@@ -1871,14 +1863,6 @@ added: v8.4.0
 The `'session'` event is emitted when a new `Http2Session` is created by the
 `Http2SecureServer`.
 
-#### Event: `'sessionError'`
-<!-- YAML
-added: v8.4.0
--->
-
-The `'sessionError'` event is emitted when an `'error'` event is emitted by
-an `Http2Session` object associated with the `Http2SecureServer`.
-
```

Comment:
I think we can't just remove `'sessionError'`, we should 1) doc-deprecated it and 2) keep emitting both.

Reply:
+1! I've moved it to `experimental` for now indicating that `'sessionError'` may be removed in the future, or do you think we should entirely deprecate it?

Reply:
That's not what experimental means. I would go with deprecated.

Reply:
I see. Sorry about that. In particular, the statement, "Non-backward compatible changes or removal may occur in any future release." regarding experimental changes in the docs had me confused a bit. I've made the changes.